vllm/entrypoints/openai/cli_args.py

```diff
@@ -130,6 +130,14 @@ def make_arg_parser(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
         "using app.add_middleware(). ")
 
     parser = AsyncEngineArgs.add_cli_args(parser)
+
+    parser.add_argument('--max-log-len',
+                        type=int,
+                        default=None,
+                        help='Max number of prompt characters or prompt '
+                        'ID numbers being printed in log.'
+                        '\n\nDefault: Unlimited')
```

Comment:
It's just to keep the same ordering of arguments as before. Since the last argument for `AsyncEngineArgs` is `--disable-log-requests`, I think it would be more appropriate to display `--max-log-len` after it.